.github/workflows/vitest.yml

```diff
@@ -0,0 +1,51 @@
+name: Vitest
```

Comment:
CircleCI please: https://www.notion.so/mui-org/code-infra-Migrate-out-of-CircleCI-42350363b7344380a9961cf9731aae31 for the final version of this effort.

Reply:
Yeah, this is just for ensuring the changes work while working on them